src/components/sidebar/__internal__/sidebar-header/sidebar-header.style.js

```diff
@@ -5,7 +5,7 @@ const SidebarHeaderStyle = styled.div`
   background-color: ${({ theme }) => theme.colors.white};
   box-shadow: inset 0 -1px 0 0 ${({ theme }) => theme.disabled.border};
   box-sizing: content-box;
-  width: 100%;
+  width: 100% - 32px;
```

Comment:
I don't think this is really fixing the problem, just covering it up with specific CSS

I think the `box-sizing: content-box;` is the issue, and it should be changed to `box-sizing: border-box;` to take the padding into account

https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
Good point 😃